packages/api/AlvTime.Business/Users/UserService.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace AlvTime.Business.Users
+{
+    public class UserService
+    {
+        private readonly IUserStorage _storage;
+
+        public UserService(IUserStorage storage)
+        {
+            _storage = storage;
+        }
+
+        public UserResponseDto CreateUser(CreateUserDto user)
+        {
+            UserQuerySearch criterias = new UserQuerySearch { Email = user.Email, Name = user.Name };
+            if (!GetUsers(criterias).Any())
```

Comment:
Her kunne du evt også gjort samme type validering som for CreateCustomer. Dersom en bruker med denne eposten eller dette navnet finnes fra før, throw new ValidationException, else create user.